auth_supplier/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': "Auth Supplier",
     'category': 'Portal',
-    'version': '9.0.2.0.0',
+    'version': '10.0.1.0.0',
```

Comment:
No, it's correct as it is. The subversion should be reset on each new Odoo version.